docs/odh/oci-model-storage.md

```diff
@@ -8,15 +8,228 @@ which also explains how to deploy models from OCI images.
 
 This page offers a guide similar to the upstream project documentation, but
 focusing on the OpenDataHub and OpenShift characteristics. To demonstrate
-how to create an OCI container image, the publicly available [MobileNet v2-7
-model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
-is used. This model is in ONNX format.
+how to create and use OCI containers, two examples are provided:
+* The first example uses [IBM's Granite-3.0-2B-Instruct model](https://huggingface.co/ibm-granite/granite-3.0-2b-instruct) 
+  available in Hugging Face. This is a generative AI model.
+* The second example uses the [MobileNet v2-7 model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+  is used. This is a predictive AI model in ONNX format.
 
-The ODH projects provides configurations for the OpenVINO model server, which
+## Creating and deploying an OCI image of IBM's Granite-3.0-2B-Instruct model
+
+IBM's Granite-3.0-2B-Instruct model is [available at Hugging Face](https://huggingface.co/ibm-granite/granite-3.0-2b-instruct).
+To create an OCI container image, the model needs to be downloaded and copied into
+the container. Once the OCI image is built and published in a registry, it can be
+deployed on the cluster.
+
+The ODH project provides configurations for the vLLM model server, which
+supports running the Granite model. Thus, this guide will use this model server
+to demonstrate how to deploy the Granite model stored in an OCI image.
+
+### Storing the Granite model in an OCI image
+
+To download the Granite model, the [`huggingface-cli download` command](https://huggingface.co/docs/huggingface_hub/guides/cli#huggingface-cli-download) can be used.
```

Comment:
maybe add a link about how to use git?

Reply:
@guimou recommended to not use `git` on the docs, because it clones the whole repository and you can end up with a very, very big image by including all git history on the image.

Reply:
yes, please consider only the third comment :)

Reply:
After re-reading both this paragraph and the next one, I'm not sure how I can improve it. IMO, the next paragraph is enough to let users know that the proposed process is ran in a container.

But if it is not clear, please suggest and I will accept the improvement.

Reply:
We could change to something like this:
```
To download the Granite model,  the [`huggingface-cli download` command](https://huggingface.co/docs/huggingface_hub/guides/cli#huggingface-cli-download) will be used by the OCI Container build process to download the model and build the final Container image.
```

As first reading it without advancing, the user will think, oh, so I need to download the cli, let me install it before proceeding (telling that because I usually do this lol)

Reply:
Thanks. Now I get much better the idea.
I changed the text.